packages/repo-tools/src/commands/api-reports/api-extractor.ts

```diff
@@ -210,7 +223,7 @@ const ALLOW_WARNINGS = [
 async function resolvePackagePath(
   packagePath: string,
 ): Promise<string | undefined> {
-  const projectRoot = resolvePath(__dirname, '..');
+  const projectRoot = resolvePath(process.cwd());
```

Comment:
Could make this a followup, but I think it's best if we find the true project root using [`findPaths`](https://github.com/backstage/backstage/blob/5bb2d0f32b4b6522f17b76283bc58f2213ad6726/packages/cli-common/src/paths.ts#L118) from `cli-common`. That way you can run this from anywhere in the repo.

Reply:
Yes, thanks I will include those on follow-up refactor